src/frontend/app/shared/components/list/list-table/table-cell-radio/table-cell-radio.component.html

```diff
@@ -1 +1 @@
-<mat-radio-button [disabled]="(dataSource.isAdding$ | async) || disable" (change)="dataSource.selectedRowToggle(row, false)" [checked]="dataSource.selectedRows?.has(dataSource.getRowUniqueId(row))"></mat-radio-button>
+<mat-radio-button [disabled]="(dataSource.isAdding$ | async) || disable" (change)="dataSource.selectedRowToggle(row)" [checked]="dataSource.selectedRows?.has(dataSource.getRowUniqueId(row))"></mat-radio-button>
```

Comment:
There's a mistake in the interface for selectedRowToggle, it should have an optional second parameter that defaults to true (see list.data.source).